data/me/retired/Owen-D-Cass-320c9bff-e9a6-4d84-ae58-f2457d92c6ad.yml

```diff
@@ -1,5 +1,5 @@
 id: ocd-person/320c9bff-e9a6-4d84-ae58-f2457d92c6ad
-name: "Owen D. Cas\xC3\xA1s"
+name: "Owen D. Cass"
```

Comment:
Did this manually, but have since updated the `inconsistent name` check to try to ignore non-english chars. Question with this one:

Why does this person have a different last name in their `family_name` vs `name`?

```yaml
name: "Owen D. Cas\xC3\xA1s"
family_name: "Cas\xE1s"
```

...playing w the people data, I started to infer that usually `name` keeps non-english characters unmodified and `family_name` switches the non-english characters to their closest english counterpart. But in this case, both fields use non-english characters, but just different ones?

Reply:
No idea, could be a mistake. There's not supposed to be any difference between the last name in the `name` field and as in the `family_name` field, at least as far as I'm aware. Likely just a result of one field being updated and not the other, etc.